### ISSUE TITLE
Agent avatars are not shown correctly

### ISSUE THREAD

**Issue by kilodilo** [10clouds/aiconsole#668]
![image](https://github.com/10clouds/aiconsole/assets/152965457/d2d1ef7e-e286-4f08-b4fa-4cf4ec622af4)
The agents are not shown correctly in the Project box

When I enter the project I can see these agents:
![image](https://github.com/10clouds/aiconsole/assets/152965457/6d6e6fa2-6f6c-499f-9052-71ffa68021f1)

So there is some misalignment

**Comment by aga-gasinska** [comment]
Now it's kind of mixed as I can see agent's avatars bur also static avatar

![Zrzut ekranu 2024-02-2 o 10 37 59](https://github.com/10clouds/aiconsole/assets/155951278/84aa2c5b-9db7-4c3e-9366-cedc503db1d1)

**Comment by udislau** [comment]
> Now it's kind of mixed as I can see agent's avatars bur also static avatar
> 
> ![Zrzut ekranu 2024-02-2 o 10 37 59](https://private-user-images.githubusercontent.com/155951278/301797908-84aa2c5b-9db7-4c3e-9366-cedc503db1d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4Njk5OTEsIm5iZiI6MTcwNjg2OTY5MSwicGF0aCI6Ii8xNTU5NTEyNzgvMzAxNzk3OTA4LTg0YWEyYzViLTlkYjctNGMzZS05MzY2LWNlZGM1MDNkYjFkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMDI4MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMmVhY2RkZWJlMGQ0YWViZGVmYWYzOWM4NTQ4NjBlNDU2ZmNhZmFiYzZlN2VmOGJkNWQ0ZDk0ZGEyMWM1MTllJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WZwbDyOivVtQ2-Pz360V-G-7i62FyAB9HjvYRGINM9Q)

This is correct behaviour. This field shows all agents in a given project, enabled and disabled.

Default agent avatar is shown, cause one of the agent has it.

**Comment by aga-gasinska** [comment]
Shouldn't it show agents that are used most often in the project instead of all of them? That's how I understand what @kilodilo wrote above 

> The project box should automatically show two agents that are mostly used in the project

**Comment by kilodilo** [comment]
Yes, what it does now, it shows disabled agents :) at least on my project it is so :D

**Comment by aga-gasinska** [comment]
But should it show them or not? If within my project disabled agents are not used should their avatars be visible?

**Comment by kilodilo** [comment]
No, we should show the most recently used agents. So only enabled agents, not disabled.

**Comment by aga-gasinska** [comment]
we need to have info about agent's status on the main page

**Comment by aga-gasinska** [comment]
it's correct now 

![Image](https://github.com/10clouds/aiconsole/assets/155951278/af04a52b-3db7-4032-9cd9-10b1ecbe6a3e)
